### PR TITLE
[DO NOT MERGE]update Collection and Data docs for node sdk 2.0.2

### PR DESCRIPTION
**Pull request by ThyeeZz**

### CODE REVIEW

File: API_Reference/milvus-sdk-node/v2.0.2/Collection/createIndex.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ new milvusClient(MILUVS_ADDRESS).collectionManager.createIndex(CreateIndexReq);
 | Parameter       | Description      | type                | required    |
 | --------------- | ---------------- | ------------------- | ----------- |
 | collection_name | Collection name  | String              | true        |
-| field_name      | Filed name       | string              | true        |
+| field_name      | Filed name       | String              | true        |
+| index_name      | Index name       | String              | false       |
 | extra_params    | index parameters | ShowCollectionsType | IndexParams |
```
Comment (nameczz) [inline API_Reference/milvus-sdk-node/v2.0.2/Collection/createIndex.md:16]: required IndexParams ? extra_params need check

File: API_Reference/milvus-sdk-node/v2.0.2/Collection/describeCollection.md [hunk 2]
```diff
@@ -20,5 +20,21 @@ new milvusClient(MILUVS_ADDRESS).collectionManager.describeCollection({
 ```
 ## Return
 ```javascript
-// create collection return
+// describe collection return
+{
+  status: { error_code: 'Success', reason: '' },
+  schema: {
+    fields: [ [Object], [Object] ],
```
Comment (nameczz) [inline API_Reference/milvus-sdk-node/v2.0.2/Collection/describeCollection.md:27]: fields can not show as Object. need more details

File: API_Reference/milvus-sdk-node/v2.0.2/Collection/describeCollection.md [hunk 3]
```diff
@@ -20,5 +20,21 @@ new milvusClient(MILUVS_ADDRESS).collectionManager.describeCollection({
...
+    name: 'my_collection',
+    description: '',
+    autoID: false
+  },
+  collectionID: '434827658485039105',
+  consistency_level: 'Session',
+  virtual_channel_names: [
```
Comment (nameczz) [inline API_Reference/milvus-sdk-node/v2.0.2/Collection/describeCollection.md:34]: remove these two for now.   
virtual_channel_names: [
    'by-dev-rootcoord-dml_6_434827658485039105v0',
    'by-dev-rootcoord-dml_7_434827658485039105v1'
  ],
  physical_channel_names: [ 'by-dev-rootcoord-dml_6', 'by-dev-rootcoord-dml_7' ],

File: API_Reference/milvus-sdk-node/v2.0.2/Collection/getCollectionStatistics.md [hunk 4]
```diff
@@ -22,4 +22,9 @@ new milvusClient(MILUVS_ADDRESS).collectionManager.getCollectionStatistics({
 ## Return
 ```javascript
 // getCollectionStatistics return
+{
+  stats: [ { key: 'row_count', value: '0' } ],
```
Comment (nameczz) [inline API_Reference/milvus-sdk-node/v2.0.2/Collection/getCollectionStatistics.md:26]: 1. status
2. data
3. stats

File: API_Reference/milvus-sdk-node/v2.0.2/Collection/createIndex.md [hunk 5]
```diff
@@ -1,35 +1,30 @@
...
 ### CreateIndexReq
-| Parameter       | Description      | type                | required    |
+| Parameter       | Description      | Type                | Required    |
 | --------------- | ---------------- | ------------------- | ----------- |
-| collection_name | Collection name  | String              | true        |
-| field_name      | Filed name       | string              | true        |
-| extra_params    | index parameters | ShowCollectionsType | IndexParams |
+| collection_name | Collection name  | String              | True        |
+| field_name      | Name of the field to create index on       | String              | True        |
+| index_name      | Name of the index to create      | String              | False       |
```
Comment (YiyunNi) [inline API_Reference/milvus-sdk-node/v2.0.2/Collection/createIndex.md:15]: What is the default value of `index_name` if it is not specified?

File: API_Reference/milvus-sdk-node/v2.0.2/Collection/describeIndex.md [hunk 6]
```diff
@@ -1,25 +1,42 @@
 # describeIndex()
-List all collections or get collection loading status.
```
Comment (YiyunNi) [inline API_Reference/milvus-sdk-node/v2.0.2/Collection/describeIndex.md:2]: Does this description really match this method? This method seems to be used to check index details, not to list collections and check their loading status...Please doublecheck.

File: API_Reference/milvus-sdk-node/v2.0.2/Collection/showCollections.md [hunk 7]
```diff
@@ -10,14 +10,14 @@ new milvusClient(MILUVS_ADDRESS).collectionManager.showCollections(ShowCollectio
 ### ShowCollectionsReq
 | Parameter       | Description         | type                | required |
 | --------------- | ------------------- | ------------------- | -------- |
-| collection_name | Collection name     | String              | true     |
+| collection_name | Collection name     | String array        | false    |
```
Comment (YiyunNi) [inline API_Reference/milvus-sdk-node/v2.0.2/Collection/showCollections.md:13]: I do not quite understand this parameter. If this method is used to list all collections, then why do the users have to provide the collection names? Also, if this method checks the loading status, then why do the users have to provide the loading status in the following parameter?

File: API_Reference/milvus-sdk-node/v2.0.2/Collection/getIndexState.md [hunk 8]
```diff
@@ -11,6 +11,8 @@ new milvusClient(MILUVS_ADDRESS).collectionManager.getIndexState(GetIndexStateRe
 | Parameter       | Description     | type   | required |
```
Comment (YiyunNi) [inline API_Reference/milvus-sdk-node/v2.0.2/Collection/getIndexState.md:11]: How is this method `getIndexState()` different from `getIndexBuildProgress()`? We need to reflect their difference in the description.

**Review by YiyunNi — CHANGES_REQUESTED** [review]
Please see the comments. Thanks!

File: API_Reference/milvus-sdk-node/v2.0.2/Collection/createIndex.md (hunk 5 above)
Comment (nameczz) [reply API_Reference/milvus-sdk-node/v2.0.2/Collection/createIndex.md:15]: undefined  in coding. For user, i think they don't need default index_name

File: API_Reference/milvus-sdk-node/v2.0.2/Collection/describeIndex.md (hunk 6 above)
Comment (nameczz) [reply API_Reference/milvus-sdk-node/v2.0.2/Collection/describeIndex.md:2]: you are right for this. May be you can copy it from pyMilvus?

File: API_Reference/milvus-sdk-node/v2.0.2/Collection/showCollections.md (hunk 7 above)
Comment (nameczz) [reply API_Reference/milvus-sdk-node/v2.0.2/Collection/showCollections.md:13]: When type is InMemory,  you can specify  which collections will return these collection's inMemory_percentages.
If type is All collection, this parameter is useless.

File: API_Reference/milvus-sdk-node/v2.0.2/Collection/getIndexState.md (hunk 8 above)
Comment (nameczz) [reply API_Reference/milvus-sdk-node/v2.0.2/Collection/getIndexState.md:11]: getIndexState just return index build finished or not.
getIndexBuildProgress will return total_rows and indexed_rows. So we can know the progress.

**Review by YiyunNi — APPROVED** [review]

**Pull request merged**